tests/pass-dep/libc/pthread-threadname.rs

```diff
@@ -199,4 +199,28 @@ fn main() {
         .unwrap()
         .join()
         .unwrap();
+
+    // Now set the name for a non-existing thread and verify error codes.
```

Comment:
```suggestion
    // Now set the name for a non-existing thread and verify error codes.
    // (FreeBSD does not have a thread ID argument.)
```

Reply:
Nope, it's not because FreeBSD having no thread ID arguments. It has it, but has no result.

Reply:
Ah okay. Whatever the reason is, please add a comment explaining it. :)

Reply:
Done.